PATCH-P0-FIX-UPSTREAM: Replacing skip-locking with skip-external-locking

Option 'skip-locking' is getting deprecated so we are replacing it in default
configuration files with option 'skip-external-locking'.

Maintainer: Michal Hrusecky <mhrusecky@suse.cz>

Index: support-files/my-huge.cnf.sh
===================================================================
--- support-files/my-huge.cnf.sh.orig
+++ support-files/my-huge.cnf.sh
@@ -25,7 +25,7 @@ socket		= @MYSQL_UNIX_ADDR@
 [mysqld]
 port		= @MYSQL_TCP_PORT@
 socket		= @MYSQL_UNIX_ADDR@
-skip-locking
+skip-external-locking
 key_buffer_size = 384M
 max_allowed_packet = 1M
 table_open_cache = 512
Index: support-files/my-large.cnf.sh
===================================================================
--- support-files/my-large.cnf.sh.orig
+++ support-files/my-large.cnf.sh
@@ -25,7 +25,7 @@ socket		= @MYSQL_UNIX_ADDR@
 [mysqld]
 port		= @MYSQL_TCP_PORT@
 socket		= @MYSQL_UNIX_ADDR@
-skip-locking
+skip-external-locking
 key_buffer_size = 256M
 max_allowed_packet = 1M
 table_open_cache = 256
Index: support-files/my-medium.cnf.sh
===================================================================
--- support-files/my-medium.cnf.sh.orig
+++ support-files/my-medium.cnf.sh
@@ -28,7 +28,7 @@ port		= @MYSQL_TCP_PORT@
 socket		= @MYSQL_UNIX_ADDR@
 # Change following line if you want to store your database elsewhere
 datadir	= /var/lib/mysql
-skip-locking
+skip-external-locking
 key_buffer_size = 16M
 max_allowed_packet = 1M
 table_open_cache = 64
Index: support-files/my-small.cnf.sh
===================================================================
--- support-files/my-small.cnf.sh.orig
+++ support-files/my-small.cnf.sh
@@ -26,7 +26,7 @@ socket		= @MYSQL_UNIX_ADDR@
 [mysqld]
 port		= @MYSQL_TCP_PORT@
 socket		= @MYSQL_UNIX_ADDR@
-skip-locking
+skip-external-locking
 key_buffer_size = 16K
 max_allowed_packet = 1M
 table_open_cache = 4
